pkg/controller/policy/policy_controller.go

```diff
@@ -697,23 +698,28 @@ func FlattenPorts(ports []securityv1alpha1.SecurityPolicyPort) ([]policycache.Ru
 			continue
 		}
 
-		begin, end, err := policycache.UnmarshalPortRange(port.PortRange)
-		if err != nil {
-			return nil, fmt.Errorf("portrange %s unavailable: %s", port.PortRange, err)
-		}
+		// Split port range to multiple port range, e.g. "22,80-82" to ["22","80-82"]
+		portRange := strings.Split(port.PortRange, ",")
+
+		for _, subPortRange := range portRange {
+			begin, end, err := policycache.UnmarshalPortRange(subPortRange)
+			if err != nil {
+				return nil, fmt.Errorf("portrange %s unavailable: %s", subPortRange, err)
```

Comment:
Since API server validate the input and wehook valid the format, is there any way hit like the "80-20" error type? In case input like: 21, 100-200, 80-50, the first two ranges configured, but the last one failed.